src/handlers.jl

```diff
@@ -138,7 +138,7 @@ function complete_request(socket, msg)
 
     codestart = find_parsestart(code, cursorpos)
     comps, positions = Base.REPLCompletions.completions(code[codestart:end], cursorpos-codestart+1)
```

Comment:
Probably need:
```jl
if VERSION < v"0.7.0-DEV.3500" # julia#25544
    import Base: REPLCompletions
else
    import REPL: REPLCompletions
end
```
above so that we can just call this as `REPLCompletions.completions` with no deprecation warning.

Reply:
Thanks, there was a similar `if VERSION` check at the top so I added the imports there.